cadctap/cadctap/core.py

```diff
@@ -406,17 +415,255 @@ def query(self, query, output_file=None, response_format='VOTable',
                         footer = '\n({} rows affected)'.format(rows)
                     print(footer, file=f)
 
-    def schema(self, table=None):
+    def schema(self, name=None):
```

Comment:
Don't see an advantage of having that. I was tempted to call the argument `resource` since the method displays the schema of a DB resource but in this case `resource` is a bit overloaded as it might be confused with the service.